modules/afuser/afuser.c

```diff
@@ -149,18 +150,20 @@ afuser_dd_queue(LogPipe *s, LogMessage *msg, const LogPathOptions *path_options)
           fd = open(line, O_NOCTTY | O_APPEND | O_WRONLY | O_NONBLOCK);
           if (fd == -1)
             {
-              msg_error("Opening tty device failed, disabling usertty() for 10 minutes",
+              msg_error("Opening tty device failed, disabling usertty() for time-reopen seconds",
                         evt_tag_str("user", _get_utmp_username(ut)),
                         evt_tag_str("line", line),
+                        evt_tag_int("time_reopen", self->time_reopen),
                         evt_tag_error("errno"));
-              self->disable_until = now + 600;
+              self->disable_until = now + self->time_reopen;
```

Comment:
questions:
1. why the default 600 is changed ?
2. if you plan to override the `time-reopen()` from global if configured, please allow local settings as well, so user could change per afuser destination.

Reply:
1. There are some problems with `time-reopen()` in general. When `GlobalConfig` is created its `time_reopen` is set to 60, and every driver will get that value. So actually it does not matter what value I put here, it will always come from the `GlobalConfig`.
2. There are other drivers, which support `time-reopen()`, but none of them support it on the driver level, only from the global `options{}` block. So I decided to not support it here, either.

So in general, I tried to follow the current `time-reopen()` logic. But your questions are valid, and I think that we should rework `time-reopen()` a bit:
1. Set `GlobalConfig`'s `time_reopen` to -1, by default.
2. Follow up this change in each driver; if they were using the `GlobalConfig` default, set their default to 60.
3. Add `KW_TIME_REOPEN` to driver level options, where the driver supports it.
4. Run tests / add new tests to check regression.

It will be a bit bigger patch, but I think this rework was due for some time. What do you think?